Cargo.toml

```diff
@@ -3,9 +3,9 @@ members = ["block-producer", "node", "proto", "rpc", "store", "utils"]
 resolver = "2"
 
 [workspace.dependencies]
-miden-crypto = { package = "miden-crypto", git = "https://github.com/0xPolygonMiden/crypto", branch = "next" }
-miden-lib = { package = "miden-lib", git = "https://github.com/0xPolygonMiden/miden-base", branch = "main" }
-miden-objects = { package = "miden-objects", git = "https://github.com/0xPolygonMiden/miden-base", branch = "main" }
+miden-crypto = { package = "miden-crypto", git = "https://github.com/0xPolygonMiden/crypto", branch = "andrew-integrate-winterfell" }
+miden-lib = { package = "miden-lib", git = "https://github.com/0xPolygonMiden/miden-base", branch = "andrew-integrate-winterfell" }
+miden-objects = { package = "miden-objects", git = "https://github.com/0xPolygonMiden/miden-base", branch = "andrew-integrate-winterfell" }
```

Comment:
maybe lets wait until this branch is merged downstream? Otherwise the changes LGTM